python/paddle/fluid/contrib/slim/quantization/post_training_quantization.py

```diff
@@ -795,7 +807,13 @@ def _update_program(self):
             activation_quantize_type=self._activation_quantize_type,
             weight_quantize_type=self._weight_quantize_type,
             quantizable_op_type=major_quantizable_op_types)
-        transform_pass.apply(graph)
+
+        for i, sub_graph in enumerate(graph.all_sub_graphs()):
+            if self._quant_blocks != -1 and isinstance(
+                    self._quant_blocks, list) and i not in self._quant_blocks:
+                continue
+            sub_graph._for_test = True
```

Comment:
add comments for why setting sub_graph._for_test = True

Reply:
Done，thx